server/pbench/bin/index-pbench

```diff
@@ -941,179 +1279,97 @@ def main(options, args):
-            #     except Exception:
-            #         print("{} {}\n".format(x["_type"], len(x["_source"].keys())))
-            if debug_time_operations: ts("pprint")
-            actions_l = []
-            tocs = 3
-            tods = 15
-            for action in actions:
-                if action['_type'] == 'pbench-run-toc-entry':
-                    tocs -= 1
-                    if tocs >= 0:
-                        actions_l.append(action)
-                elif action['_type'] == 'pbench-tool-data':
-                    tods -= 1
-                    if tods >= 0:
-                        actions_l.append(action)
-                else:
-                    actions_l.append(action)
-            print("len(actions) = {}".format(len(actions_l)))
-            pprint(actions_l, width=120)
-            if debug_time_operations: ts("Done", newline=True)
-            return 0
-
-        # returns 0 0r 1
-        if debug_time_operations: ts("es_index")
-        res =  es_index(es, actions)
-        if debug_time_operations: ts("Done", newline=True)
-        return res
+        # returns 0 or 1
+        if options.debug_time_operations: ts("es_index")
+        res = es_index(es, actions)
 
     except ConfigFileNotSpecified as e:
         print(e, file=sys.stderr)
-        return 2
+        res = 2
 
     except ConfigFileError as e:
         print(e, file=sys.stderr)
-        return 3
+        res = 3
 
     except UnsupportedTarballFormat as e:
         print("Unsupported Tarball Format - no metadata.log: ", e, file=sys.stderr)
-        return 4
+        res = 4
 
     except BadDate as e:
         print("Bad Date: ", e, file=sys.stderr)
-        return 5
+        res = 5
 
     except filenotfounderror as e:
         print("No such file: ", e, file=sys.stderr)
-        return 6
+        res = 6
 
     except BadMDLogFormat as e:
         print("The metadata.log file is curdled in tarball: ", e, file=sys.stderr)
-        return 7
+        res = 7
 
     except MappingFileError as e:
         print(e, file=sys.stderr)
-        return 8
+        res = 8
 
     except TemplateError as e:
-        print(repr(err), file=sys.stderr)
-        return 9
+        print(repr(e), file=sys.stderr)
+        res = 9
 
     except SosreportHostname as e:
         print("Bad hostname in sosreport: ", e, file=sys.stderr)
-        return 10
+        res = 10
+
+    except tarfile.TarError as e:
+        print("Can't unpack tarball into {}: {}".format(_tmpdir, e), file=sys.stderr)
+        res = 11
 
     # this is Spinal Tap
     except Exception as e:
         print("Other error", e, file=sys.stderr)
         import traceback
         print(traceback.format_exc())
-        return 11
+        res = 12
+
+    finally:
+        if options.debug_time_operations: ts("Done", newline=True)
+
+    # status codes: these are used by pbench-index to segregate tarballs into
```

Comment:
@ndokos, @chaitanyaenr, can you double check this comment specifically to make sure it makes sense?

I wonder if we should just move the comments into the except clauses themselves instead of separately.  Thoughts?